.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "editor.tokenColorCustomizations": {
+    "comments": "",
+    "textMateRules": []
+  }
+}
```

Comment:
I am wondering about the vscode settings in your project??? Why is it a part of your project?

Reply:
We don't know how it got into the project file, sorry. We removed it. ✌️